### PR TITLE
#[176153381] Integrate TravisCI, coveralls and code climate with readme badge

### PR DESCRIPTION
**Pull request by unadine**
## What does this PR do?
Integrate TravisCI, coveralls and code climate with readme badge

## Description of Task to be completed?
adding badges to the readme

## Any background context you want to provide?
Integrate TravisCI, coveralls and code climate with readme badge

## What are the relevant pivotal tracker stories?
Integrate TravisCI, coveralls and code climate with readme badge

### CODE REVIEW

**Review by James-nyagatare — COMMENTED** [review]
@unadine  what about the badge of Travis and coveralls?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<a href="https://codeclimate.com/github/atlp-rwanda/runners-bn-backend/maintainability"><img src="https://api.codeclimate.com/v1/badges/4d9c365b8ad4284f03c3/maintainability" /></a>
+<<<<<<< HEAD
```
Comment (harerakalex) [inline README.md]: @unadine kindly remove unnecessary stuff from this file.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+=======
+
+[![Build Status](https://travis-ci.org/atlp-rwanda/runners-bn-backend.svg?branch=ch-integration-176153381)](https://travis-ci.org/atlp-rwanda/runners-bn-backend)
+
+[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/runners-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/runners-bn-backend?branch=develop)
+>>>>>>> chore(badges):Adding badges
```
Comment (harerakalex) [inline README.md]: This also is not needed

**Review by harerakalex — CHANGES_REQUESTED** [review]
@unadine nice job

**Review by harerakalex — APPROVED** [review]

**Pull request merged**